v03_pipeline/lib/tasks/base/base_update_task.py

```diff
@@ -2,12 +2,13 @@
 import luigi
 
 from v03_pipeline.lib.misc.io import write
-from v03_pipeline.lib.model import DatasetType, Env, ReferenceGenome
+from v03_pipeline.lib.model import DatasetType, Env, ReferenceGenome, SampleType
```

Comment:
do we need to pass this parameter around anymore?

Reply:
I'd honestly prefer to just keep it around.  I think this would be the second (or third) time I've added and removed it... and we 100% need it passed in when we do the sample type validation.